### PR TITLE
3D Tiles - Specify glTF up-axis in tileset.json

### PR DESCRIPTION
**Pull request by lilleyse**
For #3241 
For https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/175
Merge after #5004 

Adds support for a `gltfUpAxis` property in `tileset.json` that may be `"X"`, `"Y"`, or `"Z"`, and passes the axis to `Model` (building upon https://github.com/AnalyticalGraphicsInc/cesium/pull/4962). By default the `gltfUpAxis` is `"Y"` since the glTF spec states that models are y-up.

### CODE REVIEW

**Comment by pjcozzi** [comment]
@pmconne this is for design models.

**Comment by pjcozzi** [comment]
Looks OK to me, can you open a PR for the spec?  I'm not 100% sure this should be a top-level property (per tile seems like overkill especially since external tilesets could be used), but I wonder if it belongs under `asset` or if we will end up grouping properties like this in another top-level object property.

**Comment by lilleyse** [comment]
`asset` seems like a good place. I'll update this and the spec.

**Comment by pjcozzi** [comment]
@lilleyse let me know when this is ready.

**Comment by lilleyse** [comment]
@pjcozzi Could you look at the spec PR first - https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/182

If that seems ok I'll update here.

**Comment by pjcozzi** [comment]
@lilleyse please bump when ready.

**Comment by lilleyse** [comment]
This is ready.

File: Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js [hunk 1]
```diff
@@ -159,6 +159,8 @@ define([
                                 if (picked instanceof Cesium3DTileFeature) {
                                     that._feature = picked;
                                     that._updateStats(true);
+                                } else {
+                                    that._feature = undefined;
```
Comment (lilleyse) [inline Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js]: I noticed that features weren't being unselected due to changes in https://github.com/AnalyticalGraphicsInc/cesium/pull/5081, so this is a fix.

**Pull request merged**